docs/en/observability/apm/ingest-pipelines.asciidoc

```diff
@@ -65,6 +65,5 @@ If you prefer more guidance, see one of these tutorials:
 
 * <<apm-filters-ingest-pipeline>> — Learn how to obfuscate passwords stored in the `http.request.body.original` field.
 * <<apm-data-stream-rerouting>> — Learn how to reroute APM data to user-defined APM data streams.
-* {fleet-guide}/data-streams-pipeline-tutorial.html[Transform data with custom ingest pipelines] — A basic Elastic integration tutorial where you learn how to add a custom field to incoming data.
```

Comment:
This is a shame, as the tutorial is good, but some instructions do not apply to ES plugin

Reply:
Tracking next steps for that tutorial in https://github.com/elastic/observability-docs/issues/4314.